CHANGELOG.md

```diff
@@ -5,7 +5,6 @@ Monitor the release status by regions at [AKS-Release-Tracker](https://releases.
 
 ### Announcements
  
-* Starting on March 21, 2023, traffic to k8s.gcr.io will be redirected to registry.k8s.io, following the [community announcement](https://kubernetes.io/blog/2023/03/10/image-registry-redirect/).
 * Docker container runtime will be retired for Windows nodepools on May 1, 2023. After docker container runtime is retired, you may remain on existing deployed instances but scaling operations will fail, nodepool creation will fail, and you will be out of support. Follow the detailed steps [in our documentation](https://learn.microsoft.com/azure/aks/learn/quick-windows-container-deploy-cli) to upgrade to containerd.
```

Comment:
nit for tense: "Docker container runtime will be retired for Windows nodepools on May 1, 2023" -> "Docker container runtime has been retired for Windows nodepools on May 1, 2023"